cli/main.rs

```diff
@@ -413,18 +427,37 @@ async fn bundle_command(
 
   let global_state = GlobalState::new(flags)?;
 
-  let bundle_result = tsc::bundle(
+  // TODO(bartlomieju): probably should respect --quiet flag
+  eprintln!("Bundling {}", module_specifier.to_string());
+
+  let output = tsc::bundle(
     &global_state,
     compiler_config,
-    module_name,
+    module_specifier,
     maybe_import_map,
-    out_file,
     global_state.flags.unstable,
   )
-  .await;
+  .await?;
 
   debug!(">>>>> bundle END");
-  bundle_result
+
+  let output_string = fmt::format_text(&output)?;
+
+  if let Some(out_file_) = out_file.as_ref() {
+    eprintln!("Emitting bundle to {:?}", out_file_);
+
+    let output_bytes = output_string.as_bytes();
+    let output_len = output_bytes.len();
+
+    deno_fs::write_file(out_file_, output_bytes, 0o666)?;
+    // TODO(bartlomieju): do we really need to show this info? (it doesn't respect --quiet flag)
```

Comment:
It should respect the quiet flag.  I think it is good for it to express what it is doing though.  Especially with creating a file, to give some sort of information about that file.